### PR TITLE
[zh]Add 2022-05-16-volume-populators-beta.md

### LINKED ISSUE #33586: [zh]This page blog/_posts/2022-05-16-volume-populators-beta.md has no Chinese version
**This is a Feature Request**

This page blog/_posts/2022-05-16-volume-populators-beta.md has no Chinese version

**What would you like to be added**

Translate Chinese version for blog/_posts/2022-05-16-volume-populators-beta.md has no Chinese version

**Why is this needed**

This page blog/_posts/2022-05-16-volume-populators-beta.md has no Chinese version

**Comments**
/language zh
/assign

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fix #33586

Translate Chinese version for blog/_posts/2022-05-16-volume-populators-beta.md has no Chinese version

### CODE REVIEW

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+[lib-volume-populator](https://github.com/kubernetes-csi/lib-volume-populator) 库，
+只提供一个小型控制器，以及一个能够连接到卷并向卷写入适当数据的 pod 镜像。
+
+<!--
+Individual populators can be extremely generic such that they work with every type
+of PVC, or they can do vendor specific things to rapidly fill a volume with data
+if the volume was provisioned by a specific CSI driver from the same vendor, for
+example, by communicating directly with the storage for that volume.
+-->
+单个填充器是非常通用的，它们可以与所有类型的 PVC 一起工作，
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:226]:
```suggestion
单个填充器非常通用，它们可以与所有类型的 PVC 一起使用，
```

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+## How can I learn more?
+-->
+## 我怎样才能了解更多？
+
+<!--
+The enhancement proposal,
+[Volume Populators](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1495-volume-populators), includes lots of detail about the history and technical implementation
+of this feature.
+-->
+增强提案
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:240]:
```suggestion
增强提案，
```

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+只提供一个小型控制器，以及一个能够连接到卷并向卷写入适当数据的 pod 镜像。
+
+<!--
+Individual populators can be extremely generic such that they work with every type
+of PVC, or they can do vendor specific things to rapidly fill a volume with data
+if the volume was provisioned by a specific CSI driver from the same vendor, for
+example, by communicating directly with the storage for that volume.
+-->
+单个填充器是非常通用的，它们可以与所有类型的 PVC 一起工作，
+如果卷是由来自同一供应商的特定 CSI 驱动程序供应的，
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:227]:
```suggestion
或者如果卷是来自同一供应商的特定 CSI 驱动程序供应的，
```

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+它们可以执行特定于供应商的操作以快速用数据填充卷，例如，通过通信直接使用该卷的存储。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:228]:
```suggestion
它们可以执行供应商特定的的操作以快速用数据填充卷，例如，通过通信直接使用该卷的存储。
```

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+-->
+## 我怎样才能了解更多？
+
+<!--
+The enhancement proposal,
+[Volume Populators](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1495-volume-populators), includes lots of detail about the history and technical implementation
+of this feature.
+-->
+增强提案
+[卷填充器](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1495-volume-populators)
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:241]:
```suggestion
[卷填充器](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1495-volume-populators)，
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Comment by zhangxyjlu** [comment]
I've revised all of your reviews@[howieyuen](https://github.com/howieyuen)，thank you.

**Comment by zhangxyjlu** [comment]
I've revised all of your reviews@[tengqm](https://github.com/tengqm)，thank you.

**Pull request merged**